### PR TITLE
Feature/2711 eusettings ui

### PR DESCRIPTION
**Pull request by melloskitten**
This PR implements the UI for ticket [#2667](https://jira.itc.sap.com/browse/EXPOSUREAPP-2667). Please note that this PR __excludes__ the PR that handles an empty list of countries, which is handled by this ticket here [#2557](https://jira.itc.sap.com/browse/EXPOSUREAPP-2557).

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/EUSettingsViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+	// MARK: - Attributes.
+
+	let countryModels: [CountryModel]
+
+	// MARK: - Initializers.
+
+	init(countries availableCountries: [Country]) {
+		self.countryModels = availableCountries
+			.sorted { $0.localizedName <  $1.localizedName }
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/EUSettingsViewModel.swift:44]:
```suggestion
			.sorted { $0.localizedName.localizedCompare($1.localizedName) == .orderedAscending }
```

Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/EUSettingsViewModel.swift:44]: Using localizedCompare ensures that umlautes and similar language features are sorted correctly :)

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -25,6 +25,7 @@
 
 "Common_ENError5_Description" = "Fehler 5: Es ist etwas schiefgelaufen. Ihr Risiko konnte leider nicht ermittelt werden. Wir arbeiten an einer Lösung für dieses Problem.";
 
+"Common_ENError11_Description" = "Fehler 11: Es ist etwas schiefgelaufen. Ihr Risiko konnte leider nicht ermittelt werden. Bitte starten Sie Ihr Handy neu und versuchen Sie es morgen erneut.";
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:28]: I guess this is doubled up by mistake

File: src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/Cells/FlagIconCell.swift [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import UIKit
+
+/// - NOTE: The implementation may raise a 'kCFRunLoopCommonModes' warning that is a known UIKit bug: https://developer.apple.com/forums/thread/132035
+class FlagIconCell: UITableViewCell {
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/Cells/FlagIconCell.swift:24]: I have basically the same requirement in my story for the submission flow and just used the IconCell, maybe we could also use it here? Or maybe you can convince me to use this one as well 😜

**Review by nickguendling — CHANGES_REQUESTED** [review]

**Review by janetback — APPROVED** [review]
UA reviewed

File: src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/Cells/FlagIconCell.swift (hunk 3 above)
Comment (melloskitten) [reply src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/Cells/FlagIconCell.swift:24]: Ah, perfect! In that case I will change it to the IconCell. :^)

File: src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/EUSettingsViewController.swift [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+	}
+
+	// MARK: Data Source setup methods.
+
+	private func setupDataSource() {
+		client?.supportedCountries(completion: { [weak self] result in
+			switch result {
+			case .failure:
+				// TODO: We have not defined any behaviour yet for a failed country list download.
+				print("The country list could not be loaded.")
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/ENSetting/EUSettings/EUSettingsViewController.swift:65]:
```suggestion
				logError("The country list could not be loaded.")
```

**Review by marcussc — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

**Pull request merged**